src/bors/handlers/trybuild.rs

```diff
@@ -167,21 +168,36 @@ pub(super) async fn command_try_cancel<Client: RepositoryClient>(
         return Ok(());
     };
 
-    if let Err(error) = cancel_build_workflows(repo, db, &build).await {
-        tracing::error!(
-            "Could not cancel workflows for SHA {}: {error:?}",
-            build.commit_sha
-        );
-    }
-
-    db.update_build_status(&build, BuildStatus::Cancelled)
-        .await?;
-
-    tracing::info!("Try build cancelled");
+    match cancel_build_workflows(repo, db, &build).await {
+        Err(error) => {
+            tracing::error!(
+                "Could not cancel workflows for SHA {}: {error:?}",
+                build.commit_sha
+            );
+            db.update_build_status(&build, BuildStatus::Cancelled)
+                .await?;
 
-    repo.client
-        .post_comment(pr_number, "Try build cancelled.")
-        .await?;
+            repo.client
+                .post_comment(pr_number, "There was an error cancelling try build!")
+                .await?
+        }
+        Ok(workflow_ids) => {
+            db.update_build_status(&build, BuildStatus::Cancelled)
+                .await?;
+            tracing::info!("Try build cancelled");
+
+            let mut try_build_cancelled_comment = r#"Try build cancelled.
+Cancelled workflows:"#
+                .to_string();
+            for id in workflow_ids {
+                let url = repo.client.get_workflow_url(id).await;
```

Comment:
This doesn't need to be an `async` method, let's just keep it sync for simplicity.